test/libsolidity/util/TestFileParser.cpp

```diff
@@ -542,6 +563,11 @@ void TestFileParser::Scanner::scanNextToken()
 				advance();
 				selectToken(Token::Arrow);
 			}
+			else if (peek() == ' ')
+			{
+				advance();
+				selectToken(Token::Effect, scanEffectString());
+			}
 			else
 				selectToken(Token::Sub);
```

Comment:
Ah, we already have it. Why do we need to make a distinction between `Sub` and `Effect`?

Reply:
I changed that now to only use `Sub`.

Reply:
There was a problem regarding just using `Sub`. I noticed it after changing the code to only use `Sub`. The problem is that calls that don't not return any value got ambiguous, the effect got then partially parsed as a parameter. The easiest solution was just to just keep that `SideEffect` token.